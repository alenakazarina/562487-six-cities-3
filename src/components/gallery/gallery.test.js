import React from 'react';
import renderer from 'react-test-renderer';
import Gallery from './gallery';

const imageSources = [
  `image1`, `image2`, `image3`
];

describe(`Gallery`, () => {
  it(`should render Gallery correctly`, () => {
    const tree = renderer.create(
        <Gallery images={imageSources} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
