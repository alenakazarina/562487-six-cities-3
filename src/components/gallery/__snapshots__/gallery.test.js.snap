// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery should render Gallery correctly 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  >
    <div
      className="property__image-wrapper"
      style={
        Object {
          "width": "259px",
        }
      }
    >
      <img
        alt="Interior photo"
        className="property__image"
        src="image1"
      />
    </div>
    <div
      className="property__image-wrapper"
      style={
        Object {
          "width": "259px",
        }
      }
    >
      <img
        alt="Interior photo"
        className="property__image"
        src="image2"
      />
    </div>
    <div
      className="property__image-wrapper"
      style={
        Object {
          "width": "259px",
        }
      }
    >
      <img
        alt="Interior photo"
        className="property__image"
        src="image3"
      />
    </div>
  </div>
</div>
`;
